src/screens/AddNewItemScreen/index.tsx

```diff
@@ -64,9 +89,24 @@ export const AddNewItemScreen: FC = () => {
     });
   }, [onPressBack, navigation]);
 
-  const onPress = () => {
+  const onPressAdd = () => {
     addItem(text);
```

Comment:
why is `addItem` function moved from this functtion? you use it only here!

Reply:
i fix it